GREEDY/DataManagers/IImageFormatting.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GREEDY.DataManagers
+{
+    public interface IImageFormatting
+    {
+        Bitmap Format(Bitmap bitmap);
```

Comment:
Maybe the method should be called FormatImage, because Format is a bit ambiguous.